x-pack/plugin/eql/qa/mixed-node/src/test/java/org/elasticsearch/xpack/eql/qa/mixed_node/EqlSearchIT.java

```diff
@@ -222,6 +314,23 @@ private String randomEvent() {
         return expectedResponse;
     }
 
+    private void assertMultiValueFunctionQuery(Set<String> availableFunctions, Set<String> testedFunctions, Request request,
+        RestClient client, String functionName, String query, int[] ids) throws IOException {
+        // trivial checks on the validity of functions being used
+        assertTrue("Function [" + functionName + "] not found in the list of functions " + availableFunctions,
+            availableFunctions.contains(functionName));
+        assertTrue("Function [" + functionName + "] not found in query [" + query + "].",
+            query.contains(functionName + "(") || query.contains(functionName + "~("));
```

Comment:
FMI: are these needed, given the last check of `testMultiValueFields()`? (reminded me of this anti-["defensive programming"](https://github.com/elastic/elasticsearch/pull/66022#discussion_r557676794) recommendation)

Reply:
Fair point. I'll remove them.